src/plugins/disqus.js

```diff
@@ -26,7 +26,6 @@ function install(hook, vm) {
     div.style = `width: ${main.clientWidth}px; margin: 0 auto 20px;`;
     dom.appendTo(dom.find('.content'), div);
 
-    // eslint-disable-next-line
     window.disqus_config = function () {
```

Comment:
I think we should leave this error as a warning, so we can remember to fix it. Globals are bad, and that's what we'll want to move away from with ESM and custom elements.

Reply:
What error are you referring to?

As far as globals go, they aren't "bad" for our product as it currently ships. When we are ready to promote an ESM build we can worry about globals.

Reply:
Ah ok. I didn't run the code, I assumed the line was removed because it no longer errored.

Globals are "bad" because we know we want to move away from them. Prism.js being a global is annoying when it comes to converting to ESM for example. I know we'll keep them for now.

I wouldn't mind at all having warnings for globals (and not failing on warnings, only on errors).

But if you are fine hunting down globals later without a warning helping you, then I can live with it.

Reply:
> Globals are "bad" because we know we want to move away from them. Prism.js being a global is annoying when it comes to converting to ESM for example.

Understood. Here's my $0.02:

1. Lint warnings are not TODO items. They are intended to be an ephemeral notification that a *potential* issue has been found. The goal is to address the issue and remove the need for the warning sooner than later because--theoretically, at least--people don't want to see lint warnings unnecessarily or for a prolonged period of time.
2. Removing globals is future work we have discussed but nobody has committed to doing or proposed a timeline for other than "after v5". We don't want to repeatedly "warn" developers about existing code that is not an issue today, won't be for the near future (v5), and may never be because as over time the project direction may change, feature/enhancement requirement may change, or perhaps nobody commits to doing the "future" work that justified the warning in the first place.
3. If we want to tag certain lines or blocks of code, I think comments are a better way of doing this. This is already being done in our repo with the `/* html */` comment. We could continue this precedent using a `/* global */` which would provide the same level of assistance with "hunting down globals later".

If/when someone commits to removing globals (perhaps as part of a larger effort) with a reasonable timeline, I think it would be beneficial to enable lint rules that warned about the use of globals 1) on the branch where the development is taking place or 2) in `main` after the work is merged to prevent accidental global usage in the future. Until then, I don't think the use of globals in v4 and v5 is anything contributors need to be "warned" about.